services/brig/src/Brig/User/Search/Index.hs

```diff
@@ -217,7 +217,7 @@ mkUserQuery (review _TextId -> self) t q =
           [ termQ "suspended" "1", -- [Note: suspended]
             termQ "_id" self
           ],
-        ES.boolQueryMustMatch = [optionallySearchWithinTeam t]
+        ES.boolQueryMustMatch = [optionallySearchWithinOwnTeam t]
```

Comment:
@fisx It doesn't matter if it is _own_ team, the team is specified as `t`

Reply:
de106f7